### ISSUE TITLE
Game crashes on start

### ISSUE THREAD

**Issue by NezzleGod** [Crendgrim/DynamicCrosshair#51]
Using Fabric 1.20

https://pastebin.com/antP5gFA - Crash report

The game crashes when i start it, i have cloth config installed, latest version

**Comment by Crendgrim** [comment]
This looks like it cannot read the config file. I think I forgot to double check whether old config files can be parsed properly.

Until I fix it, you should be able to delete the dynamiccrosshair.json5 config file (and then re-configure the mod). Apologies for that.

**Comment by Crendgrim** [comment]
I may be wrong because I cannot replicate it. Can you upload your config file?

**Comment by Crendgrim** [comment]
Ah, I only tested the 1.19 → 1.20 transition, not 1.18 → 1.20.

I pushed version 7.0.1 that fixes this and makes the old config readable. I'll have to investigate a better solution than crashing on malformed configs, but this works for now.

Just as an aside, cloth-config isn't required anymore. To configure the mod graphically, install yet-another-config-lib instead (but it will work just fine without).
